### PR TITLE
Clarify TTL and T 

### LINKED ISSUE #51: Clarify the usage of T for EKTKey in Section 2.3
These functions have the property that D(K, E(K, P)) = ( P
   concatenated with optional padding) for all values of K and P.  Each
   cipher also has a limit T on the number of times that it can be used
   with any fixed key value.  The EKTKey MUST NOT be used more that T
   times.

>>> Where is the counter kept to enforce T?  It does not seem to be
>>> one of the parameters associated with the SPI for outbound
>>> processing.  Also, what does an implementation do if it reaches
>>> the limit imposed by T?

### PR DESCRIPTION
**Pull request by fluffy**
Fixes #51, #53, #55

### CODE REVIEW

File: srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml [hunk 1]
```diff
@@ -936,14 +954,12 @@ SRTP packets                 <------->      SRTP packets
...
-
-      <t>This draft is a cut down version of
-      draft-ietf-avtcore-srtp-ekt-03 and much of the text here came
-      from that draft.</t>
+      David Benham, Yi Cheng, Lakshminath Dondeti, Kai Fischer,
+      Nermeen Ismail, Paul Jones, Eddy Lem, Jonathan Lennox, Michael
+      Peck, Rob Raymond, Sean Turner, Magnus Westerlund, and Felix
+      Wyss for fruitful discussions, comments, and contributions to
+      this document.</t>
+
```
Comment (fluffy) [inline srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml]: All I did is add Sean Turner and sort by last name

Comment (suhasHere) [inline srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml]: Missing Russ in the kist

File: srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml [hunk 2]
```diff
@@ -439,9 +442,22 @@ EKTField = FullEKTField / ShortEKTField / ExtensionEKTField
             period of time, so that out of order packets can be
             processed as well as packets sent during the time keys are
             changing.</t>
-          </section>
-        </section>
+
+            <t> When receiving a new EKTKey, implications need to use
+            the ekt_ttl to create a time after which this key can not
```
Comment (suhasHere) [inline srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml]: Implications- implementations

Comment (suhasHere) [inline srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml]: Can not - cannot

File: srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml [hunk 3]
```diff
@@ -439,9 +442,22 @@ EKTField = FullEKTField / ShortEKTField / ExtensionEKTField
...
+            be used and they also need to create a counter that keeps
+            track of how many times the keys has been used to encrypt
+            data to ensure it does not exceed the T value that
+            cipher. If either of theses limits are exceeded, the key
```
Comment (suhasHere) [inline srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml]: Value that cipher - value for that cipher

**Comment by suhasHere** [comment]
Changes look good , except for few typos .

**Comment by paulej** [comment]
Nothing to add beyond Suhas' comments.

**Pull request merged**